code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -575,11 +571,6 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 		else
 			shielding_powered = FALSE
 
-	///function used to check the limit of the canisters and also set the amount of damage that the canister can receive, if the heat and pressure are way higher than the limit the more damage will be done
-	if(!excited && !protected_contents && (our_temperature > heat_limit || our_pressure > pressure_limit))
-		take_damage(clamp((our_temperature/heat_limit) * (our_pressure/pressure_limit), 5, 50), BURN, 0)
-		excited = TRUE
-
```

Comment:
If my reading of the code is right, this isn't necessary since all cases of a can's gasmix being changed is accounted for and will already set the can to process.

Reply:
Try heating a canister of nitrogen over the limit with the shielding on, the turn it off and see that nothing will happen.
That's because the canister doesn't react to the temperature change, especially once settled to a certain temperature and without ongoing reactions.
There is a need to check here if the pressure and temperature are too high and to set the canister to excited if needed

Reply:
Oh right, forgot about that case. I suppose it might be better to have the toggle throw the can into process. Will do this later.

Reply:
e5582aa096f15169c909cd54e91c77d43bc83cfe should address this. Thank you.